lib/forms/find_support.yml

```diff
@@ -0,0 +1,113 @@
+_
+find_support:
+- key: are_you_out_of_employment
+  group: not_in_work
+  type: radio
+  required: true
+  options:
```

Comment:
There seems to be quite a lot of repetition here, especially in defining all the options again. It seems like there's more chance of human error? 

There also seems to be an option defined above (yes_but_someone_else) which doesn't appear in this file - is that correct?